Library/Homebrew/cleaner.rb

```diff
@@ -89,6 +91,14 @@ def clean_dir(d)
         next
       elsif path.extname == ".la"
         path.unlink
+      elsif path.basename.to_s == "perllocal.pod"
+        # Both this file & the .packlist one below are completely unnecessary
+        # to package & causes pointless conflict with other formulae. They are
+        # removed by Debian, Arch & MacPorts amongst other packagers as well.
+        # The files are created as part of installing any Perl module.
+        path.unlink
+      elsif path.basename.to_s == ".packlist" # Hidden file, not file extension!
```

Comment:
Yeah, I thought about that but I wanted to include the comment about it being a hidden file to clarify I wasn't being an idiot about not using the `extname` further up, and I wasn't sure if that would be confusing clumped in with the same comments about `perllocal.pod`.